### DISCUSSION TITLE
关于插件中的下载的快捷键(CTRL+AFT+D)修改为(F8)做个小小调查

### DISCUSSION THREAD

**Discussion by jacksonjim** [jacksonjim/keil-assistant#20] [Polls]
关于插件中的下载的快捷键(CTRL+AFT+D)修改为(F8)做个小小调查，目前有部分人需求调整修改默认快捷键为F8，所以在此做个小小调查下基本情况

**Comment by DJZ1992** [comment]
你好，能否提个建议。使用cubemx生成工程，MDK的工程文件在MDK-ARM文件夹下（在工程目录的下一层）。当前使用vscode打开工程目录，插件可以找到MDK-ARM下的工程文件，并在MDK-ARM下新建.vscode文件夹.
问题是打开的工程文件为MDK-ARM的上一级目录,而该目录下没有.vscode,所以一些跳转会失灵,且还有一些报错.需要手动复制.vscode到工程目录下,才能正确跳转

**Reply by jacksonjim** [reply]
目前这个问题可能与另一个功能冲突，多个项目文件时都生成同一个.vscode，会出错的，除非去除了这个多个项目的支持

**Reply by DJZ1992** [reply]
万分感谢。我下载了你的源码，把vscode的生成路径往上挪了一级，本地安装使用。基本上能凑合了着用了。
再次感谢
